tests/integration/test_yarn.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from pathlib import Path
+
+import pytest
+
+from . import utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "test_params",
+    [
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-yarn-berry.git",
+                ref="76131443f7472916b316954c957259ee4cee97a7",
```

Comment:
In the READMEs, could you explain which features the test cases cover?

Reply:
The commit history might be helpful for that https://github.com/cachito-testing/cachi2-yarn-berry/commits/offline-cache